style.css

```diff
@@ -0,0 +1,371 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+html {
+    background-color: #FFFFFF;
+} 
+  
+body {
+    margin: 0 auto;
+    width: 100%; 
+    max-width: 800px;
+    font-family: 'Roboto Condensed', sans-serif;
+    /* background-color: #FFFFFF; */
+    padding: 0px 30px;
+  }
+
+header {
+    font-family: 'Merienda', cursive, century-gothic, Verdana;
+    background-color: #69027B;
+    color: #FFFFFF;
+    padding: 30px 20px 30px 35px;
+}
+
+.menu {
+    padding: 5px;
+    /* float: right; */
+    margin: 0px 0px 40px 0px;
+    display: inline;
+    text-decoration: none;
+    color: white;
+    text-decoration: none;
+}
+
+.menu_item {
+    max-width: 15%;
+    padding: 10px 15px 10px 15px;
+    display: inline;
+    margin: 5px;
+    background-color: #97D8B2;
+    font-family: 'Roboto Condensed', sans-serif;
+    /* color: white;
+    text-decoration: none; */
+}
+
+a.menu_item {
+    text-decoration: none;
+}
+
+a.menu_item:hover {
+    text-decoration: underline;
+}
+
+.social_item {
+    max-width: 30%;
+    float: right;
+    padding: 5px;
+    margin: 0px 20px;
+    font-size: 36px;
+    background-color: #FFFFFF;
+}
+
+.fa-google {
+    background: #dd4b39;
+    color: white;
+}
+
+.fa-linkedin {
+    background: #327BB0;
+    color: white;
+}
+
+img {
+    max-width: 100%;
+    margin: 0 auto;
+    padding: 5px;
+}
+
+.photo {
+  width: 80%;
+  margin: 0 auto;
+  clear: both;
+}
+
+.about {
+    line-height: 2rem;
+    font-family: 'Roboto Condensed', sans-serif;
+    padding: 10px 15px 10px 15px;
+}
+
+.aboutme {
+    padding: 0px 15px 0px 15px;
+    line-height: 2rem;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+
+.project1 {
```

Comment:
For this and the next few lines, you should consider abstracting this a bit more. You could make a `project` class that contains the CSS that is the same between the projects and then have separate classes for the differences. I don't know if this makes sense without the HTML, but it might look like:

```css
.project {
    font-family: 'Roboto Condensed', sans-serif;
    font-weight: bold;
    border: 1px solid black;
    padding: 5px;
}

.project1 {
    margin: 0 auto;
    clear: both;
}

.project2 {
    margin: 10px 0px;
}
```

Reply:
This is very helpful to keep in mind! To make sure I am understanding correctly: basically re-arrange and organize the classes so no only they, but also the CSS can be clearer and more concise, as well as more easily adjusted?